katello/hooks/pre_validations/30-mongo_storage_engine.rb

```diff
@@ -35,6 +35,11 @@ def disk_space
     fail_and_exit 'Upgrading is not supported on remote MongoDB database connections' unless mongo_host == 'localhost:27017'
   end
 
+  # Make sure MongoDB is running before start of engine upgrade
+  Kafo::Helpers.log_and_say :info, 'Checking to see if MongoDB is running before executing storage upgrade.'
+  `systemctl status rh-mongodb34-mongod`
+  execute('foreman-maintain service start --only rh-mongodb34-mongod') unless $?.success?
```

Comment:
Do we need to check status? Can we just always try to start mongod to be on the safe side?